### ISSUE TITLE
provide a standalone mode for tox similar to the one used by py.test

### ISSUE THREAD

**Issue by pytoxbot** [tox-dev/tox#95]
- Bitbucket: https://bitbucket.org/hpk42/tox/issue/95
- Originally reported by: @ssbarnea
- Originally created at: 2013-04-17T16:49:07.934

`py.test` can generate a [standalone script](http://pytest.org/latest/goodpractises.html#create-a-py-test-standalone-script) you can run the tests without having to install py.test.

This is important for enabling the usage on continuous integration systems where you do not have root access to the running machine (which is usually a clean machine, sometimes even spawned on demand),

In order to overcome this, I tried to setup a virtual environment and install tox in it but this doesn't seem to work well, as tox will try to create other virtual environments itself.

**Comment by obestwalter** [comment]
@ssbarnea is there still need for something like this and would you like to follow up on it?

**Comment by ssbarnea** [comment]
At this moment is just a **nice to have** feature and I am not able to commit on implementing it, meanwhile I changed job, still using tox a lot but not really blocked by this.

Even if you close it we can always reopen it if it gets enough votes.

**Comment by obestwalter** [comment]
@ssbarnea thanks for your quick answer. Let's leave it open then as enhancement and revisit later :)

**Comment by RonnyPfannschmidt** [comment]
we also killed the feature in pytest - i propose closing the issue,
these days tools exist to make this possible, people that care can start with a pr
